<ui version="4.0">
 <class>Regist</class>
 <widget class="QWidget" name="Regist">
  <property name="geometry">
   <rect><x>0</x><y>0</y><width>680</width><height>620</height></rect>
  </property>
  <property name="minimumSize">
   <size><width>680</width><height>620</height></size>
  </property>
  <property name="windowTitle"><string>注册</string></property>

  <layout class="QVBoxLayout" name="mainLayout">
   <property name="leftMargin"><number>20</number></property>
   <property name="topMargin"><number>20</number></property>
   <property name="rightMargin"><number>20</number></property>
   <property name="bottomMargin"><number>20</number></property>

   <!-- 顶部大标题：楷体 + 放大字号 + 深色 -->
   <item>
    <widget class="QLabel" name="lbTitle">
     <property name="objectName"><string>headline</string></property>
     <property name="text"><string>工业现场远程专家支持系统</string></property>
     <property name="alignment"><set>Qt::AlignHCenter|Qt::AlignVCenter</set></property>
     <property name="margin"><number>8</number></property>
     <property name="styleSheet">
      <string notr="true">color:#0f172a; font-family:&quot;KaiTi&quot;,&quot;STKaiti&quot;,&quot;Kaiti SC&quot;,&quot;KaiTi_GB2312&quot;,&quot;Noto Serif SC&quot;,serif; letter-spacing:1px;</string>
     </property>
     <property name="font">
      <font>
       <pointsize>28</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="minimumHeight">
      <number>60</number>
     </property>
    </widget>
   </item>

   <item>
    <widget class="QFrame" name="card">
     <property name="frameShape"><enum>QFrame::NoFrame</enum></property>
     <property name="frameShadow"><enum>QFrame::Plain</enum></property>
     <layout class="QVBoxLayout" name="cardLayout">
      <property name="leftMargin"><number>24</number></property>
      <property name="topMargin"><number>12</number></property>
      <property name="rightMargin"><number>24</number></property>
      <property name="bottomMargin"><number>12</number></property>
      <property name="spacing"><number>12</number></property>

      <item><widget class="QComboBox" name="cbRole"/></item>
      <item><widget class="QLineEdit" name="leUsername">
        <property name="placeholderText"><string>账号</string></property>
      </widget></item>
      <item><widget class="QLineEdit" name="lePassword">
        <property name="placeholderText"><string>密码</string></property>
        <property name="echoMode"><enum>QLineEdit::Password</enum></property>
      </widget></item>
      <item><widget class="QLineEdit" name="leConfirm">
        <property name="placeholderText"><string>确认密码</string></property>
        <property name="echoMode"><enum>QLineEdit::Password</enum></property>
      </widget></item>

      <item>
       <spacer name="spacer_between_confirm_and_buttons">
        <property name="orientation"><enum>Qt::Vertical</enum></property>
        <property name="sizeHint">
         <size><width>20</width><height>8</height></size>
        </property>
       </spacer>
      </item>

      <item>
       <layout class="QVBoxLayout" name="btnsLayout">
        <item><widget class="QPushButton" name="btnRegister">
          <property name="text"><string>立即注册</string></property>
        </widget></item>
        <item><widget class="QPushButton" name="btnBack">
          <property name="text"><string>返回登录</string></property>
        </widget></item>
       </layout>
      </item>

     </layout>
    </widget>
   </item>

   <item>
    <spacer name="spacer_bottom">
     <property name="orientation"><enum>Qt::Vertical</enum></property>
     <property name="sizeHint"><size><width>20</width><height>20</height></size></property>
    </spacer>
   </item>

  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
